src/utils/ValidationHelper.ts

```diff
@@ -28,4 +33,59 @@ export class ValidationHelper {
 		});
 	}
 
+	private async validateUserData(data: string | undefined, errmsg: string, logger: Logger): Promise<boolean> {
+		let isValid = true;
+		if (data == null || data === undefined || data.trim().length === 0) {
+			logger.info("UserInfo missing: ", errmsg);
+			isValid = false;
+		}
+		return isValid;
+	}
+
+	async validateUserInfo(session: ISessionItem, logger: Logger): Promise<boolean> {
+		let isValid = true;
+		if (!await this.validateUserData(session.full_name, "Full Name is missing", logger) ||
+			!await this.validateUserData(session.date_of_birth, "Date of Birth is missing", logger) ||
+			!await this.validateUserData(session.document_selected, "Document selection type is missing", logger) ||
+			!await this.validateUserData(session.date_of_expiry, "Expiry Date is missing", logger)) {
+			isValid = false;
+		}
+		return isValid;
+	}
+
+	async eventToSubjectIdentifier(jwtAdapter: KmsJwtAdapter, event: APIGatewayProxyEvent): Promise<string> {
+		const headerValue = event.headers.authorization ?? event.headers.Authorization;
+		if (headerValue === null || headerValue === undefined) {
+			throw new AppError( "Missing header: Authorization header value is missing or invalid auth_scheme", HttpCodesEnum.BAD_REQUEST);
+		}
+		const authHeader = event.headers["Authorization"] as string;
```

Comment:
could constants be moved out into Constants.ts pls

Reply:
Done.